test/Rewards.t.sol

```diff
@@ -0,0 +1,133 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.23;
+
+import "forge-std/Test.sol";
+import "forge-std/console.sol";
+import "../src/NODL.sol";
+import "../src/Rewards.sol";
+import "openzeppelin-contracts/contracts/utils/cryptography/MessageHashUtils.sol";
+
+contract RewardsTest is Test {
+    NODL nodlToken;
+    Rewards rewards;
+    address recipient;
+    uint256 oraclePrivateKey;
+
+    function setUp() public {
+        recipient = address(1);
```

Comment:
```suggestion
        recipient = vm.addr(1);
```

Reply:
We don't need to know or keep trace of the recipient's private key in this test